i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md

```diff
@@ -50,102 +66,319 @@ sans utiliser de connexion Internet directe pour gérer leur base d'installation
 
-### Comment puis-je tester le Plugin et que signifient les options des commandes ?
+## Comment puis-je tester le plugin et que signifient les options des commandes ?
 
-Une fois le Plugin installé, vous pouvez tester celui-ci directement en ligne de commande depuis votre collecteur Centreon avec l'utilisateur *centreon-engine*:
+Une fois le plugin installé, vous pouvez tester celui-ci directement en ligne
+de commande depuis votre collecteur Centreon en vous connectant avec
+l'utilisateur **centreon-engine** (`su - centreon-engine`). Vous pouvez tester
+que le connecteur arrive bien à superviser une ressource en utilisant une commande
+telle que celle-ci (remplacez les valeurs d'exemple par les vôtres) :
 
 ```bash
 /usr/lib/centreon/plugins/centreon_cisco_ssms_restapi.pl \
-      --plugin=apps::cisco::ssms::restapi::plugin \
-      --mode=licenses \
-      --hostname='myipaddress' \
-      --client-id='myapiclientid' \
-      --client-secret='myapiclientsecret' \
-      --account='1234abc-56de-78fg-90hi-1234abcdefg' \
-      --filter-counters='status' \
-      --filter-license-name='mylicence'
-      --critical-license-status='%{status} !~ /in compliance/i' \
-      --verbose
+	--plugin=apps::cisco::ssms::restapi::plugin \
+	--mode=licenses \
+	--hostname=10.0.0.1 \
+	--client-id='' \
+	--client-secret='' \
+	--port='' \
+	--proto=''  \
+	--account= \
+	--filter-counters= \
+	--filter-license-name='' \
+	--warning-license-status='' \
+	--critical-license-status='%{status} !~ /in compliance/i' \
+	--warning-usage='' \
+	--critical-usage='' \
+	--warning-usage-free='' \
+	--critical-usage-free='' \
+	--warning-usage-prct='' \
+	--critical-usage-prct='' \
+	--verbose
 ```
 
-La commande ci-dessus contrôle le statut des licences Cisco SSMS (```--mode=licences```) nommées *mylicence* (```--filter-licences-name='mylicence'```).
-Cette licence appartient au compte *1234abc-56de-78fg-90hi-1234abcdefg* (```--account='1234abc-56de-78fg-90hi-1234abcdefg'```). 
-
-Cette commande déclenchera une alarme CRITICAL si le statut de la licence est différent de *in compliance* (```--critical-license-status='%{status} !~ /in compliance/i'```).
-
-Toutes les options et leur utilisation peuvent être consultées avec le paramètre ```--help``` ajouté à la commande:
+La commande devrait retourner un message de sortie similaire à :
 
 ```bash
-/usr/lib/centreon/plugins//centreon_cisco_ssms_restapi.pl --plugin=apps::cisco::ssms::restapi::plugin 
---mode=licences --help
+OK: licenses are ok | '*accounts*~*licenses*#licenses.usage.count'=;;;0;total'*accounts*~*licenses*#licenses.free.count'=;;;0;total'*accounts*~*licenses*#licenses.usage.percentage'=%;;;0;total
```

Comment:
is it a good example?

Reply:
Is it the mode selected the problem ? Do you prefer alerts mode ?
Or is-it the ouput the problem ? In this case its looks regular for me comparing them with recent doc as Network>Teldat-SNMP for exemple : 
```
OK: All memory usages are ok | '*memory*#memory.usage.bytes'=B;;;0;total'*memory*#memory.free.bytes'=B;;;0;total'*memory*#memory.usage.percentage'=%;;;0;100
```

Reply:
I just thought that =;;;0; didn't illustrate the purpose of the mode very well, but maybe I'm wrong